sklearn/impute/tests/test_knn.py

```diff
@@ -237,7 +233,6 @@ def test_knn_imputer_verify(na):
 
 @pytest.mark.parametrize("na", [np.nan, -1])
 def test_knn_imputer_one_n_neighbors(na):
-
```

Comment:
Could you revert this, and the other single line deletions, to preserve edit history via git blame?